### ISSUE TITLE
This issue serve as a discussion for features that will be in the oh-my-dev plugin

### ISSUE THREAD

**Issue by Oghenebrume50** [MLH-Fellowship/0.0.1-Oh-My-Dev#1]
Starting with the most basic, so we can iterate quickly
- Searching
 > We want to be able to search through articles using the title and maybe the author's name

- Post 
 > We want users to be able to make a post from the command line  too 
 The post can either be made from an already existing file or typed in using the users' favourite text editor 

@OPARA-PROSPER @AlbertoPdRF  what other features do you want to see in this plugin? Please feel free to edit this issue then after that we can break it into separate issues and assign to ourselves

**Comment by AlbertoPdRF** [comment]
I would add a `tag` option on the search feature too, but other than that the first proposed functionalities LGTM. I think we can start with that and, if we are done soon enough, think about other useful stuff we could add.

**Comment by OPARA-PROSPER** [comment]
Searching articles via keywords, tags, users (probably an author you anticipate their content)

**Comment by AlbertoPdRF** [comment]
How would the via keywords part work? I don't remember seeing that option on the API documentation.

**Comment by Oghenebrume50** [comment]
Yes the keyword is not there, I think if we have tags then keywords are not really needed
